auto_py_to_exe/web/js/i18n.js

```diff
@@ -219,6 +226,7 @@ const translationMap = {
         tr: 'Ayarlar',
         th: 'การตั้งค่า',
         fr: 'Paramètres',
+        hi:'सेटिंग्स',
```

Comment:
There are a few styling issues in this file. Could you format the file using prettier? [The steps can be found here](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/CONTRIBUTING.md#style-guide).

Reply:
This file still need the auto-formatted run on it